src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -66,23 +73,137 @@ const styles = StyleSheet.create({
 });
 
 export interface AnonymousInputMessageProps {
-  onSendButtonClicked: (message: string) => void;
+  onSendButtonClicked: (message: string, attachments: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `onSendButtonClicked` now takes an additional parameter `attachments`. Make sure all calls to this function are updated accordingly.
```diff
- onSendButtonClicked: (message: string) => void;
+ onSendButtonClicked: (message: string, attachments: any) => void;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->